python/whylogs/api/logger/experimental/multi_dataset_logger/multi_dataset_rolling_logger.py

```diff
@@ -209,13 +224,14 @@ def __init__(
         self._schema: Optional[DatasetSchema] = schema
         self._store_list: List[ProfileStore] = []
 
-        if write_schedule.cadence == TimeGranularity.Second:
-            raise Exception("Minimum write schedule is five minutes.")
+        if write_schedule is not None:
+            if write_schedule.cadence == TimeGranularity.Second:
+                raise Exception("Minimum write schedule is five minutes.")
 
-        if write_schedule.cadence == TimeGranularity.Minute and write_schedule.interval < 5:
-            raise Exception("Minimum write schedule is five minutes.")
+            if write_schedule.cadence == TimeGranularity.Minute and write_schedule.interval < 5:
+                raise Exception("Minimum write schedule is five minutes.")
 
-        self._timer = FunctionTimer(write_schedule, self.flush)
+            self._timer = FunctionTimer(write_schedule, self.flush)
```

Comment:
consider adding a log statement for when there is no schedule? It could be logged as a warning if you aren't in a notebook environment.

Reply:
Can do in follow up, not blocking experimental module changes here.